site/resources/js/enhancement.js

```diff
@@ -989,6 +989,53 @@ function enhanceAll() {
 	);
 }
 
+// Focus Indicator Classes
+var NAVIGATION_KEYS = [
+	'Tab',
+	'ArrowUp',
+	'ArrowRight',
+	'ArrowDown',
+	'ArrowLeft',
+	'Home',
+	'End',
+	'PageUp',
+	'PageDown',
+	'Enter',
+	' ',
+	'Escape',
+
+	/* IE9 and Firefox < 37 */
+	'Up',
+	'Right',
+	'Down',
+	'Left',
+	'Esc'
+];
+
+var keyboardFocus = false;
+
+function onKeydownHandler(event) {
+	if (event.ctrlKey || event.altKey || event.metaKey || NAVIGATION_KEYS.indexOf(event.key) === -1) {
+		return;
+	}
+	keyboardFocus = true;
```

Comment:
This will need to be invalidated at some point - I'm seeing keyboardFocus being added for any click event as well.

Reply:
Good call! I've added some logic to set it to false on mousedown events and also added some focusout logic to remove the classes when they are no longer needed.